src/main/java/com/apeshko/javaspring/dao/TicketDao.java

```diff
@@ -0,0 +1,10 @@
+package com.apeshko.javaspring.dao;
+
+import com.apeshko.javaspring.model.User;
+
+import java.util.List;
+
+public interface TicketDao<T> extends Dao<T> {
+    List<T> getBookedTicketsByUserId(long userId, int pageSize, int pageNum);
+    List<T> getBookedTicketsByEventId(long eventId, int pageSize, int pageNum);
```

Comment:
better to provide overridden methods to get not paginated data as well, currently we force client to use pagination